build/pause/Makefile

```diff
@@ -122,14 +122,17 @@ bin/wincat-windows-${ARCH}: windows/wincat/wincat.go
 
 container: .container-${OS}-$(ARCH)
 .container-linux-$(ARCH): bin/$(BIN)-$(OS)-$(ARCH)
-	docker buildx build --pull --output=type=${OUTPUT_TYPE} --platform ${OS}/$(ARCH) \
+	docker buildx build --provenance=false --sbom=false --pull --output=type=${OUTPUT_TYPE} --platform ${OS}/$(ARCH) \
```

Comment:
does this make any change on the linux image?

Reply:
when were these flags added? are we breaking users on older versions?

Reply:
The flags were added between buildx plugin v0.9.x and v0.10.x and unfortunately the default behavior was also changed to build a manifest list by default.
Adding these flags should produce images will produce images with the same structure and OCI artifact types as the previously built / published pause images

These changes will break if uses are using buildx plugin v0.9.x but the current pause image build is also broken now (if `make all-push` is run)

cc @jeremyrickard too (he's familiar with these changes)

Reply:
when running the all-push should we add a check to validate the buildx version?